runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +578,27 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *toolCursor = strstr(MAPPING_J9NAME(j9vm_args, xdumpIndex), ":tool") + 1;
+				if (toolCursor) {
```

Comment:
Please use an explicit comparand, e.g. `NULL != toolCursor`.  Remember to put an rvalue on the left of equality and inequality operators.